server/src/main/java/org/opensearch/search/SearchHits.java

```diff
@@ -124,6 +164,23 @@ public SearchHits(StreamInput in) throws IOException {
         collapseValues = in.readOptionalArray(Lucene::readSortValue, Object[]::new);
     }
 
+    public SearchHits(byte[] in) throws IOException {
```

Comment:
Lets assert or throw an exception if Protobuf is not enabled? 
If not we will end up in NullPointerException.

Reply:
Done.